packages/dnb-design-system-portal/src/shared/menu/SidebarMenu.tsx

```diff
@@ -277,62 +248,93 @@ function ListItem({
 
   const params = {}
 
-  if (active) {
+  if (isActive) {
     params['aria-current'] = true
   }
 
   return (
-    <li
-      className={classnames(
-        'dnb-sidebar-menu',
-        `l-${level}`,
-        active && 'is-active', // use anchor hover style
-        inside && 'is-inside',
-        status ? `status-${status}` : null,
-        className
-      )}
-      ref={ref}
-      style={
-        {
-          '--delay': `${
-            nr && nr < 20 ? nr * 12 : 0 // random(1, 160)
-          }ms`,
-        } as React.CSSProperties /* Casting to allow css variable in JSX inline styling */
-      }
-    >
-      <Link
-        to={to}
-        onClick={() => {
-          closeMenu()
-        }}
+    <>
+      <li
         className={classnames(
-          'dnb-anchor',
-          'dnb-anchor--no-underline',
-          'dnb-anchor--no-radius',
-          'dnb-anchor--no-hover',
-          icon && graphics[icon] ? 'has-icon' : null
+          'dnb-sidebar-menu',
+          `l-${level}`,
+          isActive && 'is-active', // use anchor hover style
+          isInsideActivePath && 'is-inside-active-path',
+          isInsideActiveCategory && !isInsideActivePath && 'is-inside',
+          status ? `status-${status}` : null,
```

Comment:
```suggestion
          status && `status-${status}`,
```

Would this give the same result?
Reason for suggesting it is just to "follow the same style/way" as the lines above https://github.com/dnbexperience/eufemia/pull/2120/files#diff-80240cf1f7728b7dff84548e0d34eb958ce8b4ce677e7fd8ffa320a7c83241fdR261-R263

Reply:
`false` is just fine – we use false a lot inside classNames – so I would rather use it because the `null` return is unnecessary.